Gemfile

```diff
@@ -55,8 +54,11 @@ group :assets do
   gem 'coffee-rails', '~> 4.0.0'
 end
 
+group :production do
+    gem 'pg'
+end
 # Use ActiveModel has_secure_password
-gem 'bcrypt', '~> 3.1.7'
```

Comment:
the `~>` means to lock the version to all but the last number. like `3.1.X`. That way you can still pick up bug fixes but still lock the gem version. We don't generally care about locking gems in the test and developmen groups though.